6.0/6.0.0a4/skeleton/docker-entrypoint.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -e
+
+if [ -z "${PIP_PARAMS}" ]; then
+  PIP_PARAMS="--use-deprecated legacy-resolver"
```

Comment:
The legacy resolver is not needed anymore with the pip 22 that we are pinning.

Reply:
Done. For the moment I set the vars to "", just in case we need them again at some point.